Marlin/src/module/planner.cpp

```diff
@@ -2350,7 +2350,7 @@ bool Planner::_populate_block(
   block->steps_per_mm = steps_per_mm;
   uint32_t accel;
   #if ANY(LIN_ADVANCE, FTM_HAS_LIN_ADVANCE)
-    bool use_adv_lead = false;
+    bool use_advance_lead = false;
```

Comment:
This causes a naming collision with the one of the planner. I suggest you undo unrelated changes